src/key.rs

```diff
@@ -66,7 +66,7 @@ pub const ONE_KEY: SecretKey = SecretKey([0, 0, 0, 0, 0, 0, 0, 0,
 /// A Secp256k1 public key, used for verification of signatures
 #[derive(Copy, Clone, PartialEq, Eq, Debug, Hash)]
 #[repr(transparent)]
-pub struct PublicKey(ffi::PublicKey);
+pub struct PublicKey(pub(crate) ffi::PublicKey);
```

Comment:
Is this needed? Where do we use access the internal representation of the key?

Reply:
answered above

Reply:
https://github.com/rust-bitcoin/rust-secp256k1/pull/306/commits/aa6bdaff3c7554ba018cceccf3ffcae8b40bf831#diff-d25941070387cc0b48c0525480137107aad8f631a8d8c4cd9e8d0f3a456eb6ccR480